adm_program/modules/lists/lists_show.php

```diff
@@ -731,7 +736,26 @@
                     $content = $arrListValues[$member[$sqlColumnNumber]];
                 }
             }
-
+            elseif ($column->getValue('lsc_special_field') === 'mem_approved')
+            {
+                // Assign Integer to Language strings
+                if ((int) $content === 1)
```

Comment:
uh. ok. But we should implement this also. Not in this feature request but soon.